middleware/proxy/upstream.go

```diff
@@ -197,6 +198,16 @@ func parseBlock(c *caddyfile.Dispenser, u *staticUpstream) error {
 			}
 
 			u.ex = newGoogle("", boot) // "" for default in google.go
+		case "grpc":
```

Comment:
Well, I moved the .proto into CoreDNS. That way coredns-grpc will depend on coredns instead of the other way around. It also means that anyone can implement a server (middleware or otherwise) using that .proto that is in CoreDNS. So it's not directly tied to what we're doing at Infoblox with coredns-grpc.